### ISSUE TITLE
[Accessibility] Sidebar toggle is not focusable

### ISSUE THREAD

**Issue by gurmukhp** [ampproject/ampstart#569]
https://ampstart.com/templates/article.amp#amp=1

Tester has observed that when tabbing across the page, the focus indicator is not visible when highlighting the menu icon. Please note that while focus indicator is observed on other elements, focus indicator could be improved to increase visibility for native users with visual impairments. 

This issue relates to WCAG 2.1 guideline: 2.4.3.Focus Order and 3.2.2 On Input
